acceptance/tests/base/dsl/structure_test.rb

```diff
@@ -73,7 +73,7 @@
 
       fail "#confine did not skip test but should have."
     rescue Beaker::DSL::Outcomes::SkipTest => e
-      assert_match(/No suitable hosts found with {:platform=>"test"}/, e.message, "#confine raised an unexpected skip_test")
+      assert_match(/No suitable hosts found with.*platform.*"test"}/, e.message, "#confine raised an unexpected skip_test")
```

Comment:
Wouldn't it at least keep the `{` from before?
```suggestion
      assert_match(/No suitable hosts found with {.*platform.*"test"}/, e.message, "#confine raised an unexpected skip_test")
```

And I guess it's `:platform=>` vs `platform:` so can we capture it as follows?
```suggestion
      assert_match(/No suitable hosts found with {(:platform=>|platform:)"test"}/, e.message, "#confine raised an unexpected skip_test")
```